lib/cext/preprocess.rb

```diff
@@ -8,7 +8,7 @@
 
 ARGF.each do |line|
   if line.include?('rb_scan_args')
-    puts line.gsub(/rb_scan_args\((\w+), (\w+), \"(.*)\", /) {
+    puts line.gsub(/\srb_scan_args\((\w+), (\w+), \"(.*)\", /) {
```

Comment:
`\b` might be better if for some reason there is another character in front (e.g.: none or `(`).